app/models/invenio_rdm_record_converter.rb

```diff
@@ -126,13 +126,23 @@ def invenio_metadata(gf)
   end
 
   def resource_type(digitalhub_subtype)
-    irdm_subtype = DH_RESOURCE_TYPES[digitalhub_subtype]
-    irdm_type = IRDM_RESOURCE_TYPES[irdm_subtype]
+    irdm_types = DH_IRDM_RESOURCE_TYPES[digitalhub_subtype]
 
-    {
-      "type": irdm_type,
-      "subtype": irdm_subtype
-    }
+    if irdm_types && irdm_types[1]
+      {
+        "type": irdm_types[0],
+        "subtype": irdm_types[1]
+      }
+    elsif irdm_types # only Dataset has no subtype
```

Comment:
Thanks for the comment here. I glossed over it at first and was looking for when this branch would run lol.